Cesium3DTilesSelection/include/Cesium3DTilesSelection/TilesetOptions.h

```diff
@@ -328,6 +329,11 @@ struct CESIUM3DTILESSELECTION_API TilesetOptions {
    * If no ellipsoid is set, Ellipsoid::WGS84 will be used by default.
    */
   CesiumGeospatial::Ellipsoid ellipsoid = CesiumGeospatial::Ellipsoid::WGS84;
+
+  /**
+   * @brief Headers to attach to requests made for this tileset.
```

Comment:
To make this a little clearer:

```suggestion
   * @brief HTTP headers to attach to requests made for this tileset.
```